packages/api/lib/pdrHelpers.js

```diff
@@ -17,6 +18,61 @@ function generateShortPAN(disposition) {
   );
 }
 
+/**
+ * Get list of input granules from execution
+ *
+ * @param {string} executionArn - execution arn
+ * @returns {object[]} list of granules
+ */
+async function getGranulesFromExecution(executionArn) {
+  const excObj = await getExecution({
+    prefix: process.env.stackName,
+    arn: executionArn,
+  });
+  return excObj.originalPayload.granules;
+}
+
+const granulesFileCount = (granules) => granules.reduce((sum, { files }) => sum + files.length, 0);
+
+/**
+ * Generate Long PAN message
+ *
+ * @param {object|string[]} executions - List of workflow executions
+ * @returns {string} the PAN message
```

Comment:
This is an async function, it returns a Promise<string> 

```suggestion
 * @returns {Promise<string>} the PAN message
```